### PR TITLE
added support for cpHastySpace

### LINKED ISSUE #1: Add cpHastySpace support
Chipmunk 7.0 has multi-threaded implementation of solver. It also has additional performance on ARMs (makes use of NEON SIMD). Activating it is very simple: just call cpHastySpaceNew instead of cpSpaceNew and cpHastySpaceStep instead of cpSpaceStep (and perhaps set number of threads because NEON isn't very useful at a current state of Crystal). I think I can make a PR, but never tried it, so it will take some time.

### PR DESCRIPTION
**Pull request by konovod**
fixes #1 
I've no idea wy CI fails.

Does this change adds some indirection? specs on my computer passes (i've made space hasty by default), but at least on my linux `threads` can't be set to more than 2, so profit is minimal.

### CODE REVIEW

File: src/chipmunk/debug_draw.cr [hunk 1]
```diff
@@ -166,7 +166,7 @@ module CP
         end
 
         if @flags.draw_collision_points?
-          arbiters = space.@space.arbiters
+          arbiters =space.@space.value.arbiters
```
Comment (oprypin) [inline src/chipmunk/debug_draw.cr]: Spacing is off

**Review by oprypin — CHANGES_REQUESTED** [review]
The commit has "Author: Your Name <you@example.com>", that's not good

**Comment by oprypin** [comment]
I will merge this some time next week, if that's OK

**Comment by konovod** [comment]
i think i've messed up this crap, so i'll create another PR

**Pull request closed without merging**